### PR TITLE
named-inputs

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Named Inputs](https://trello.com/c/06hoxDxY/79-named-inputs)

File: operators/src/mock/mock_point_source.rs [hunk 1]
```diff
@@ -55,35 +54,38 @@ impl VectorOperator for MockPointSource {
...
+        }
+        .boxed())
     }
 }
 
+pub struct InitializedMockPointSource {
+    result_descritor: VectorResultDescriptor,
+    points: Vec<Coordinate2D>,
+}
+
```
Comment (jdroenner) [inline operators/src/mock/mock_point_source.rs:73]: typo

**Review by jdroenner — CHANGES_REQUESTED** [review]

**Comment by jdroenner** [comment]
also unify many and multiple

File: operators/src/engine/mod.rs [hunk 2]
```diff
@@ -7,11 +7,13 @@ pub use execution_context::{
...
 };
 pub use operator::{
-    InitializedOperator, InitializedOperatorBase, InitializedPlotOperator,
-    InitializedRasterOperator, InitializedVectorOperator, PlotOperator, RasterOperator,
-    TypedOperator, VectorOperator,
+    InitializedOperator, InitializedPlotOperator, InitializedRasterOperator,
+    InitializedVectorOperator, PlotOperator, RasterOperator, TypedOperator, VectorOperator,
+};
+pub use operator_impl::{
+    MultipleRasterSources, MultipleVectorSources, OneVectorManyRasterSources, Operator,
```
Comment (michaelmattig) [inline operators/src/engine/mod.rs:14]: why **Multiple**RasterSources but OneVector**Many**RasterSources? 🌑

Comment (ChristianBeilschmidt) [reply operators/src/engine/mod.rs]: 7489b0f

**Pull request merged**